src/Generator/Program.fs

```diff
@@ -417,6 +417,36 @@ let specTypeToCSharpType =
         "cl_work_group_info","WorkGroupInfo"
     |] |> Map.ofArray
 
+type DummyType =
```

Comment:
Can this be made into something more meaningful.. `TypeName` or `TypeDescriptor` or something like that

Reply:
I'll change it to `DummyTypeDescriptor`